packages/admin/admin/src/common/toolbar/backbutton/ToolbarBackButton.tsx

```diff
@@ -21,6 +21,11 @@ const Root = createComponentSlot("div")<ToolbarBackButtonClassKey>({
 
         .CometAdminToolbarItem-root {
             padding: 0;
+            button {
+                margin: auto;
+                height: fit-content;
+                padding: 10px;
+            }
```

Comment:
Is this really the best solution? How about removing the [align-items: stretch](https://github.com/vivid-planet/comet/pull/2608/files#diff-b6f0a2a7b1a8191c6cd7a515c272d3c4f8e7bca71698abbe1c7c20838ac6a273R20) style instead? Also, we don't select via HTML elements in the Admin